src/coreclr/jit/block.h

```diff
@@ -671,9 +671,6 @@ struct BasicBlock : private LIR::Range
     const char* dspToString(int blockNumPadding = 0);
 #endif // DEBUG
 
-    // Type used to hold block and edge weights
-    typedef float weight_t;
-
```

Comment:
Do we want to update these to double literals?

Also we could change the MAX to be higher, though I don't think we need the extra dynamic range. So perhaps leave it as is?